iotdb-client/client-py/pom.xml

```diff
@@ -114,13 +126,16 @@
                             <resources>
                                 <resource>
                                     <directory>${basedir}/../../iotdb-protocol/thrift-datanode/target/generated-sources-python/iotdb/thrift/</directory>
-                                    <includes>
-                                        <include>**/rpc/*</include>
-                                    </includes>
```

Comment:
Will this cause error in the import code of py-client? It seems like there is no rpc root module any more.